stages/org.osbuild.coreos.squashfs.meta.json

```diff
@@ -0,0 +1,28 @@
+{
+  "summary": "CoreOS squashfs needs`.",
+  "description": [
+    "This stage implements necessary CoreOS adjustments,",
+    "including creating links and modifying the read-only flag",
+    "in OSTree configurations.",
+    "These changes are essential to prepare the system for",
+    "utilizing the squashfs stage in osbuild."
+  ],
+  "schema_2": {
+    "options": {
+      "additionalProperties": false
+    },
+    "inputs": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "tree"
+      ],
```

Comment:
From what I can tell from the code in the stage (within this PR) this is unused. 

i.e. `r = main(args["tree"], args["mounts"], args["devices"])` has no `args["inputs"]`.